assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -4,7 +4,7 @@ import type { Tests } from './abtest';
 // ----- Tests ----- //
 
 export type AnnualContributionsTestVariant = 'control' | 'annualAmountsA' | 'notintest';
-export type NewPaymentFlowTestVariant = 'control' | 'newPaymentFlow';
+export type NewFlowTestVariant = 'control' | 'newFlow';
```

Comment:
If the test isn't defined here, could move the enum to abtest.js

Reply:
actually once I moved it, I realised it was only being used in one place where it wasn't adding any value, so have ditched it